src/BsWebSocket.re

```diff
@@ -7,7 +7,7 @@ module ErrorEvent = { type t; };
 module MessageEvent = MessageEvent;
 module OpenEvent = { type t; };
 
-[@bs.new] external make: string => t = "WebSocket";
+[@bs.new] external make: (string, ~protocol: string=?, unit) => t = "WebSocket";
```

Comment:
1. Could you rename this to `protocols` and make it accept an array instead?
2. It'd be nice if the optional argument came first, as is the standard convention in Reason/OCaml. e.g.:
```reason
[@bs.new] external make: (string, ~protocols: array(string)=?, unit) => t = "WebSocket";
let make = (~protocols=?, url) => make(url, ~protocols?, ());
```

Reply:
yep, I agree with both of these! Updated as per comments